### PR TITLE
Fixed 3 issues

### LINKED ISSUE #166: Crash when UpdateChecker can't reach GitHub
Simply disconnect from the internet to reproduce

```
PixiEditor crashed on 2021.03.21 at 12:32:58

-------Crash message-------
System.Net.Http.HttpRequestException: No such host is known. (raw.githubusercontent.com:443)
-----Inner exception-----
System.Net.Sockets.SocketException: No such host is known.

-------Stack trace-------
   at System.Net.Http.ConnectHelper.ConnectAsync(Func`3 callback, DnsEndPoint endPoint, HttpRequestMessage requestMessage, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.SendAsyncCore(HttpRequestMessage request, HttpCompletionOption completionOption, Boolean async, Boolean emitTelemetryStartStop, CancellationToken cancellationToken)
   at PixiEditor.UpdateModule.UpdateChecker.GetUpdateIncompatibleVersionsAsync(String tag) in C:\Users\phili\source\repos\PixiEditor\PixiEditor.UpdateModule\UpdateChecker.cs:line 73
   at PixiEditor.UpdateModule.UpdateChecker.IsUpdateCompatible() in C:\Users\phili\source\repos\PixiEditor\PixiEditor.UpdateModule\UpdateChecker.cs:line 64
   at PixiEditor.ViewModels.SubViewModels.Main.UpdateViewModel.<CheckForUpdate>b__17_0() in C:\Users\phili\source\repos\PixiEditor\PixiEditor\ViewModels\SubViewModels\Main\UpdateViewModel.cs:line 62
   at PixiEditor.ViewModels.SubViewModels.Main.UpdateViewModel.CheckForUpdate() in C:\Users\phili\source\repos\PixiEditor\PixiEditor\ViewModels\SubViewModels\Main\UpdateViewModel.cs:line 59
   at PixiEditor.ViewModels.SubViewModels.Main.UpdateViewModel.Owner_OnStartupEvent(Object sender, EventArgs e) in C:\Users\phili\source\repos\PixiEditor\PixiEditor\ViewModels\SubViewModels\Main\UpdateViewModel.cs:line 91
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.Run()
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run()
   at PixiEditor.App.Main()
-----Inner exception-----
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|283_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.DefaultConnectAsync(SocketsHttpConnectionContext context, CancellationToken cancellationToken)
   at System.Net.Http.ConnectHelper.ConnectAsync(Func`3 callback, DnsEndPoint endPoint, HttpRequestMessage requestMessage, CancellationToken cancellationToken)
```

### PR DESCRIPTION
**Pull request by CPKreuz**
Fixes #166, #167 and #168

### CODE REVIEW

File: PixiEditor/Views/MainWindow.xaml.cs [hunk 1]
```diff
@@ -85,75 +85,6 @@ private void MainWindowStateChangeRaised(object sender, EventArgs e)
...
-            else
-            {
-                File.Delete(updateExeFile);
-            }
-        }
-
-        private void RestartToUpdate(string updateExeFile)
-        {
-            Process.Start(updateExeFile);
-            Close();
```
Comment (flabbet) [inline PixiEditor/Views/MainWindow.xaml.cs:156]: Where is this now?

Comment (CPKreuz) [reply PixiEditor/Views/MainWindow.xaml.cs:156]: It's in UpdateSubViewModel...

**Comment by flabbet** [comment]
https://github.com/PixiEditor/PixiEditor/blob/master/PixiEditor/ViewModels/SubViewModels/Main/UpdateViewModel.cs#L57-L85

this is not the same as this

https://github.com/PixiEditor/PixiEditor/blob/master/PixiEditor/Views/MainWindow.xaml.cs#L93-L119

**Review by flabbet — CHANGES_REQUESTED** [review]
Fix the installer

File: PixiEditor/ViewModels/SubViewModels/Main/FileViewModel.cs [hunk 2]
```diff
@@ -195,23 +195,11 @@ private void Open(string path)
...
-                    try
-                    {
-                        OpenDocument(path, true);
-                    }
-                    catch (CorruptedFileException ex)
-                    {
-                        MessageBox.Show(ex.Message, "Failed to open file.", MessageBoxButton.OK, MessageBoxImage.Error);
-                    }
-                }
+                NoticeDialog.Show("This pixi file uses the old file format and can't be opened", "Old file format");
```
Comment (flabbet) [inline PixiEditor/ViewModels/SubViewModels/Main/FileViewModel.cs:202]: Maybe "This .pixi file uses the old format, which is no longer supported and can't be opened." would be better

**Review by flabbet — CHANGES_REQUESTED** [review]
Shutdown PixiEditor, open `%appdata%\PixiEditor`, change max recent files to "asdasd", open PixiEditor -> crash.

Also there is always visible restart button instead of version 
image

**Comment by flabbet** [comment]
While we are it, maybe add YouTube and Donate (open collective) button to the startup window? And perhaps icons?

**Review by flabbet — CHANGES_REQUESTED** [review]
Screenshot says it all 
image

Also, Instead of "Sponsor" maybe add "Donate"

**Comment by CPKreuz** [comment]
> Screenshot says it all 
> image
> 

I see nothing wrong here

File: PixiEditor/Views/Dialogs/HelloTherePopup.xaml [hunk 3]
```diff
@@ -168,7 +168,7 @@
                         </Button.Style>
                     </Button>
                     <Button Command="{Binding OpenHyperlinkCommand}" CommandParameter="https://opencollective.com/pixieditor"
-                            Content="Sponsor">
+                            Content="Donate" ToolTip="And subscribe to PixiEditor's OnlyFans please">
```
Comment (flabbet) [inline PixiEditor/Views/Dialogs/HelloTherePopup.xaml:171]: I'll kick your ass one day

**Review by flabbet — CHANGES_REQUESTED** [review]

**Review by flabbet — APPROVED** [review]

**Pull request merged**